proposals/integration-test-cases.md

```diff
@@ -0,0 +1,103 @@
+# Integration test case suggestions
+This document gives an overview of possible integration test cases, provides some useful links and a specification how to write the docstring for a test case.
+
+
+## Useful links
+testing advice by bitcoin
+https://github.com/bitcoin/bitcoin/tree/master/test/functional#general-test-writing-advice
+
+tendermint integration tests:
+https://github.com/tendermint/tendermint/tree/master/test
+
+## How to structure a test scenario
+The following serves as a structure to describe the tests. Each integreation test should contain this description in the docstring.
+
+| Keyword               | Description               |
+|-----------------------|---------------------------|
+| Name                  | Name of the test          |
+| Startup State         | Required base settings    |
+| Test Description      | Steps to be executed      |
+| Output Specification  | Expected output           |
+
+Startup State:
```

Comment:
I'd improve the formatting here, using `###` (header 3) or something else, up to you.